### DISCUSSION TITLE
[Feature] <Delete Duplicates，Can you give me a link to view the original image? >

### DISCUSSION THREAD

**Discussion by ptsa** [immich-app/immich#10751] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Delete Duplicates，Can you give me a link to view the original image?
Or provide a large picture for comparison？
Because sometimes the detected images are not exactly the same.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This'll be in the next release
